From ae98a2baa24a329c94c6f804bf9f8bccacab59d0 Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Wed, 2 Dec 2015 19:10:53 +0200
Subject: [PATCH 0562/1240] fix: phy: a3700: extend PLL Delay and Lock-Timeout

currently proper clock refference are set at start of u-boot day (in start.S)
Until SPL also will use correct clock logic, temporarily use extended delays and timeout for PLL locking

Change-Id: Icbc95c6af773222c0d741ba9a1dc23f8bae36220
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/25558
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/phy/comphy_a3700.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/phy/comphy_a3700.h b/drivers/phy/comphy_a3700.h
index c9ddb42..2ced4e0 100644
--- a/drivers/phy/comphy_a3700.h
+++ b/drivers/phy/comphy_a3700.h
@@ -22,8 +22,8 @@
 #include <asm/arch-mvebu/comphy_hpipe.h>
 
 #define DEFAULT_REFCLK_MHZ		25
-#define PLL_SET_DELAY_US		60
-#define PLL_LOCK_TIMEOUT		10
+#define PLL_SET_DELAY_US		600
+#define PLL_LOCK_TIMEOUT		1000
 #define POLL_16B_REG			1
 #define POLL_32B_REG			0
 
-- 
1.9.1

